src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -4,74 +4,85 @@
+  public final TalonFX pivotMotor = new TalonFX(14, Constants.OperatorConstants.canivoreSerial);
   private final TalonFX leftShootMotor = new TalonFX(5, Constants.OperatorConstants.canivoreSerial);
   private final TalonFX rightShootMotor = new TalonFX(13, Constants.OperatorConstants.canivoreSerial); 
   private double targetVelocity = 0;
   private double motorVelocity = 0;
   private boolean motorsAtShootingSpeed = false;
-  private VelocityVoltage velocity = new VelocityVoltage(targetVelocity);
-  private MotionMagicVoltage motionMagicVoltage = new MotionMagicVoltage(0.1);
+  private MotionMagicVoltage motionMagicVoltage = new MotionMagicVoltage(0);
   private final String SMART_DASHBOARD_VELOCITY = "Shooter Motor Velocity";
   private final String SMART_DASHBOARD_TARGET_VELOCITY = "Shooter Motor Target Velocity";
   private final String SMART_DASHBOARD_POSITION = "Shooter Motor Position";
+  private final String SMART_DASHBOARD_TARGET_POSITION = "Shooter Motor Target Position";
 
-  public enum Position {
-    DEFAULT,
-    AMP,
-    SPEAKER;
-  }
-
-  private static Position positionOfArm = Position.DEFAULT;
+  public CANcoder wristPositionEncoder = new CANcoder(18);
+  private double targetArmPosition = 0;
+  private static double positionOfArm = 0;
 
   public void shooterInit() {
     SmartDashboard.putNumber(SMART_DASHBOARD_VELOCITY, motorVelocity);
     SmartDashboard.putNumber(SMART_DASHBOARD_TARGET_VELOCITY, targetVelocity);
-    SmartDashboard.putString(SMART_DASHBOARD_POSITION, positionOfArm.toString());
+    SmartDashboard.putNumber(SMART_DASHBOARD_POSITION, positionOfArm);
+    SmartDashboard.putNumber(SMART_DASHBOARD_TARGET_POSITION, targetArmPosition);
+
     leftShootMotor.setInverted(true);
+
+    MagnetSensorConfigs wristPositionMagnetConfigs = new MagnetSensorConfigs();
+    wristPositionMagnetConfigs.AbsoluteSensorRange = AbsoluteSensorRangeValue.Unsigned_0To1;
+    wristPositionMagnetConfigs.SensorDirection = SensorDirectionValue.CounterClockwise_Positive;
+    wristPositionMagnetConfigs.MagnetOffset = 0;  // TODO Calibrate this with a command
+    wristPositionEncoder.getConfigurator().apply(wristPositionMagnetConfigs);
+
+    FeedbackConfigs pivotConfigs = new FeedbackConfigs();
+    pivotConfigs.FeedbackRemoteSensorID = wristPositionEncoder.getDeviceID();
+    pivotConfigs.FeedbackSensorSource = FeedbackSensorSourceValue.RemoteCANcoder;
+    pivotConfigs.RotorToSensorRatio = 45 / 8;
+    pivotMotor.getConfigurator().apply(pivotConfigs);
+
+    TalonFX.optimizeBusUtilizationForAll(pivotMotor, leftShootMotor, rightShootMotor);
     stopMotors();
   }
 
   public void optomizeCan() {
     TalonFX.optimizeBusUtilizationForAll(pivotMotor, leftShootMotor, rightShootMotor);
   }
 
-  public void startMotors(double speed, Position position) {
+  public void setAngle(double angle) {
+    motionMagicVoltage = new MotionMagicVoltage(angle / 360);
+    motionMagicVoltage = motionMagicVoltage.withOverrideBrakeDurNeutral(true);
```

Comment:
Nit, I'd rather have a MotionMagicVoltage field with the fields that don't change (such as the override brake dur neutral value) as a field and then do the with for the value on that field inside the setControl call.